src/api/install.ts

```diff
@@ -521,6 +522,10 @@ async function installInContext (
     outdatedPkgs: installCtx.outdatedPkgs,
   })
 
+  ctx.pendingBuilds = ctx.pendingBuilds
```

Comment:
I don't understand, initially, you populated this array only when scripts are ignored. Now you populate it always? Why?

Reply:
My fault! I removed the if! I'll make sure we concat only when --ignore-scripts is used